sycl/test/basic_tests/kernel_info.cpp

```diff
@@ -33,6 +33,7 @@ int main() {
   program prg(q.get_context());
 
   prg.build_with_kernel_type<class SingleTask>();
+  assert(prg.has_kernel<class SingleTask>());
```

Comment:
```suggestion
  CHECK(prg.has_kernel<class SingleTask>());
```

Reply:
Hah, embarrasing! Thank you for catching this!